python/test/t_ExtremeValueCopula_std.py

```diff
@@ -0,0 +1,114 @@
+#! /usr/bin/env python
+
+from __future__ import print_function
+from openturns import *
+from math import *
+
+TESTPREAMBLE()
+RandomGenerator.SetSeed(0)
+
+try:
+    # Instanciate one distribution object
+    dim = 2
+    copula = ExtremeValueCopula(SymbolicFunction("t", "(t-0.5)^2+0.75"))
+    print("Copula ", repr(copula))
+    print("Copula ", copula)
+    print("Mean ", repr(copula.getMean()))
+    print("Covariance ", repr(copula.getCovariance()))
+
+    # Is this copula an elliptical distribution?
+    print("Elliptical distribution= ", copula.isElliptical())
+
+    # Is this copula elliptical ?
+    print("Elliptical copula= ", copula.hasEllipticalCopula())
+
+    # Is this copula independent ?
+    print("Independent copula= ", copula.hasIndependentCopula())
+
+    # Test for realization of distribution
+    oneRealization = copula.getRealization()
+    print("oneRealization=", repr(oneRealization))
+
+    # Test for sampling
+    size = 10
+    oneSample = copula.getSample(size)
+    print("oneSample=", repr(oneSample))
+
+   # Test for sampling
+    size = 10000
+    anotherSample = copula.getSample(size)
+    print("anotherSample mean=", repr(anotherSample.computeMean()))
+    print("anotherSample covariance=", repr(anotherSample.computeCovariance()))
+
+    # Define a point
+    point = Point(dim, 0.2)
+
+    # Show PDF and CDF of point
+    pointPDF = copula.computePDF(point)
+    pointCDF = copula.computeCDF(point)
+    print("Point = ", repr(point), " pdf=%.6f" %
+          pointPDF, " cdf=%.6f" % pointCDF)
+
+    # Get 50% quantile
+    quantile = copula.computeQuantile(0.5)
+    print("Quantile=", repr(quantile))
+    print("CDF(quantile)=%.6f" % copula.computeCDF(quantile))
+
+    # Get 5% quantile
+    quantile = copula.computeQuantile(0.95, True)
+    print("Quantile=", repr(quantile))
+
+    # Get 95% survival function
+    inverseSurvival = Point(copula.computeInverseSurvivalFunction(0.95))
+    print("InverseSurvival=", repr(inverseSurvival))
+    print("Survival(inverseSurvival)=%.6f" %
+          copula.computeSurvivalFunction(inverseSurvival))
+    print("entropy=%.6f" % copula.computeEntropy())
+
+    # Confidence regions
```

Comment:
could you please remove this commented block if useless ?
Btw could you add a small check for `computeConditionalCDF`, `computeConditionalQuantile` ?

Reply:
Once Sofiane's remark is taken into account this is ok to merge.

Reply:
I missed your comment. Maybe we could split the test into 2/3 tests  in order to override the Timeout?